device-types/Raritan/DSX2-16-1.yaml

```diff
@@ -0,0 +1,56 @@
+---
+manufacturer: Raritan
+model: DSX2-16 Console Server
```

Comment:
The model name appears to be `DSX2-16`
Please remove the "`Console Server`" part

Reply:
Removed

Reply:
Remove "-1" and committed file "DSX2-16.yaml"